vizro-core/hatch.toml

```diff
@@ -55,7 +55,8 @@ schema = "python schemas/generate.py {args}"
 secrets = "pre-commit run gitleaks --all-files"
 test = [
   "test-unit",
-  "test-integration"
+  "test-integration",
+  "test-js"
```

Comment:
Is it okay? @maxschulz-COL 
Do I also need to modify the `vizro-core/docs/pages/development/contributing.md` file accordingly?

Reply:
Yes please, would be great if you could update it for that `hatch` command, and ideally double check if that section is still up to date in general